src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/PassInput.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.policy
+
+data class PassInput(
```

Comment:
Reasonably minor, but should this class and filename be PSS rather than Pass?

Reply:
possibly, we can look at fixing this